.tina/collections/logos.tsx

```diff
@@ -0,0 +1,51 @@
+import type { Collection } from "tinacms";
+import { seoSchema } from "../../components/util/seo";
+import * as Schemas from "../../components/blocks";
+
+export const logosSchema: Collection = {
+  label: "Logos",
+  name: "logos",
+  path: "content/logo",
+  format: "mdx",
+  ui: {
+    router: ({ document }) => {
+      return `/logo/${document._sys.filename}`;
+    },
+  },
+  fields: [
+    // @ts-ignore
+    seoSchema,
+    {
+      type: "string",
+      label: "Header",
+      name: "header",
+    },
+    {
+      type: "object",
+      list: true,
+      name: "_body",
+      label: "Body",
+      ui: {
+        visualSelector: true,
+      },
+      templates: [...Schemas.pageBlocks],
+    },
+    {
+      type: "object",
+      name: "footer",
+      label: "Footer",
+      fields: [
+        {
+          type: "string",
+          label: "Text",
+          name: "text",
+        },
+        {
+          type: "string",
+          label: "Link",
+          name: "link",
+        },
+      ],
+    }
```

Comment:
Can you make this a `rich-text` field instead?

Reply:
Done - changed this to a rich-text field and using markdown for the link